packages/components-native/src/index.ts

```diff
@@ -2,3 +2,4 @@ export * from "./Icon";
 export * from "./Divider";
 export * from "./Typography";
 export * from "./Text";
+export * from "./ActivityIndicator";
```

Comment:
Do we need to export the ActivityIndicatorProps also?

Reply:
We have a wrapper for react-native's `ActivityIndicator` to make it use grey blue as the default color. We don't need to export `ActivityIndicatorProps` as it comes from `react-native`